resources/proxytrack.rb

```diff
@@ -2,65 +2,62 @@
 
 default_action :create
 
-property :proxy_address,      String
+property :proxy_address,      String, default: 'localhost'
 property :proxy_port,         Integer
 property :icp_address,        String, default: 'localhost'
 property :icp_port,           Integer
 property :httrack_file_paths, Array
+property :user,               String, default: 'nobody'
 
 action :create do
   run_context.include_recipe 'proxytrack::default'
 
   systemd_unit "proxytrack-#{new_resource.name}.service" do
-    content <<-EOU.gsub(/^\s+/, '')
+    content <<-EOF.gsub(/^\s+/, '')
     [Unit]
     Description=Serve #{new_resource.name} archive
 
     [Service]
     Type=simple
     Restart=always
     RestartSec=1
-    User=centos
+    User=#{user}
     ExecStart=/bin/proxytrack #{new_resource.proxy_address}:#{new_resource.proxy_port} #{new_resource.icp_address}:#{new_resource.icp_port} #{new_resource.httrack_file_paths.join(' ')}
 
     [Install]
     WantedBy=multi-user.target
-    EOU
-    action :create
+    EOF
+    action [:create, :start, :enable]
   end
 end
 
 action :start do
-  systemd_unit "proxytrack-#{new_resource.name}.service" do
+  service "proxytrack-#{new_resource.name}" do
     action :start
   end
 end
 
-action :reload do
-  systemd_unit "proxytrack-#{new_resource.name}.service" do
-    action :reload
-  end
-end
-
 action :enable do
-  systemd_unit "proxytrack-#{new_resource.name}.service" do
+  service "proxytrack-#{new_resource.name}" do
     action :enable
   end
 end
 
 action :stop do
-  systemd_unit "proxytrack-#{new_resource.name}.service" do
+  service "proxytrack-#{new_resource.name}" do
     action :stop
   end
 end
 
 action :disable do
-  systemd_unit "proxytrack-#{new_resource.name}.service" do
+  action_stop
+  service "proxytrack-#{new_resource.name}" do
     action :disable
   end
 end
 
 action :delete do
+  action_disable
```

Comment:
Instead of this, I think you should have the service resource here doing a disable, stop actions.